install/helm/open-match/config/matchmaker_config.yaml

```diff
@@ -0,0 +1,98 @@
+debug: true
+
+logging:
+  level: debug
+  format: text
+  source: true
+
+api: 
+  backend: 
+    hostname: om-backendapi
+    port: 50505
+    backoff: "[2 32] *2 ~0.33 <30"
+  frontend: 
+    hostname: om-frontendapi
+    port: 50504
+    backoff: "[2 32] *2 ~0.33 <300"
+  mmlogic: 
+    hostname: om-mmlogicapi
+    port: 50503
+  functions:
+    port: 50502
```

Comment:
Interesting, I don't recall seeing 50502 assignments anywhere. I know knative defaults to 8080, which is what we're using as well.

Reply:
This config is a copy paste of the the matchmaker_config.yaml in other parts of the repository.